pallets/randomness-beacon/src/lib.rs

```diff
@@ -281,21 +288,21 @@ pub mod pallet {
 		#[pallet::call_index(0)]
 		#[pallet::weight(<T as pallet::Config>::WeightInfo::try_submit_asig(
 			T::MaxSigsPerBlock::get().into())
-				.saturating_add(
-					T::Dispatcher::dispatch_weight(pulses.len()))
+				// .saturating_add(
+				// 	T::Dispatcher::dispatch_weight(pulses.len()))
```

Comment:
Probably hardcoding a 1 would work until the change on the pallet manager is done
```suggestion
				 .saturating_add(
				 	T::Dispatcher::dispatch_weight(1))
```